LayoutTests/platform/gtk/TestExpectations

```diff
@@ -907,6 +907,10 @@ webkit.org/b/236298 imported/w3c/web-platform-tests/css/css-ui/text-overflow-022
 # Requires support for xHE-HAAC audio codec
 webkit.org/b/239750 media/media-source/media-mp4-xhe-aac.html [ Skip ]
 
+# Changes to logical height/width computations (https://bugs.webkit.org/show_bug.cgi?id=206161)
+webkit.org/b/206161 svg/wicd/rightsizing-grid.xhtml [ Failure ]
+webkit.org/b/206161 svg/wicd/test-rightsizing-b.xhtml [ Failure ]
```

Comment:
Referencing this bug is confusing, can you file a new bug for these 2 failures?

I'm also a bit surprised this would create GTK only failures, but perhaps GTK has a different drawing path for SVG?